### DISCUSSION TITLE
`num_bag_folds=5` will be ignored when setting `use_bag_holdout=True`?

### DISCUSSION THREAD

**Discussion by Crispy13** [autogluon/autogluon#4639] [Q&A] [answered]
`num_bag_folds=5`
`use_bag_holdout=True`

I gave the two to fit(), and `predict_proba_oof()` of the model failed.

It said it used validation set, so oof prediction is not available.

So, num_bag_folds=5 was ignored in this case?

**Comment by Innixma** [comment, marked as answer]
`num_bag_folds=5` is used, but `predict_proba_oof` probably only supports scenarios when `use_bag_holdout=False`. The oof technically exists, but I'd need to edit the existing logic to make it work in the case when `use_bag_holdout=True`.
